recipes/pytest-logging/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pytest-logging" %}
+{% set version = "2015.11.4" %}
+{% set sha256 = "cec5c85ecf18aab7b2ead5498a31b9f758680ef5a902b9054ab3f2bdbb77c896" %}
+
+package:
+  name: {{ name|lower }}
+  version: "2015.11.4"
+
+source:
+  fn: {{ name|lower }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 3
+
+requirements:
+  build:
+    - python
+    - certifi >=2016.9.26
```

Comment:
I didn't see `certifi` listed as a dependency. Is it required both here and in the `run` section?